### PR TITLE
Updating config

### PR DESCRIPTION
**Pull request by yunjchoi**
Updated the config file to show Cluster Management as its own category in AKS Diagnostics and to change the placeholder text for the search bar.

### CODE REVIEW

**Comment by yunjchoi** [comment]
@nmallick1, we are going to surface Cluster Management detector which resides in Customer category on AppLens to customers. I set the id to Customer and name to Cluster Management. Is this the right approach for showing Cluster Management as the category name?

**Comment by nmallick1** [comment]
@yunjchoi , nope, this will not work. The category array is merely to identify a category, even if there is a single detector under the category, the id and name in the file are matched against the category in applens, not the detector name

**Comment by yunjchoi** [comment]
@nmallik1 I misphrased my question. Would this map the customer-facing category to Customer category in AppLens, but show the name as Cluster Management?

**Comment by yunjchoi** [comment]
I'll update the config once the category name is updated in AppLens.

**Comment by yunjchoi** [comment]
@nmallick1, category name is now updated to match AppLens.

File: AngularApp/projects/app-service-diagnostics/src/armResourceConfig/microsoft.containerservice/managedclusters/config.json [hunk 1]
```diff
@@ -1,32 +1,24 @@
...
-		}
-	]
+    "categories": [
+        {
+            "id": "Cluster Insights",
+            "name": "Cluster Insights",
+            "description": "Is your cluster experiencing failures or unresponsiveness? Investigate and discover issues that may cause your cluster to no longer be manageable.",
+            "keywords": [ "Failures", "Node Readiness", "Node Health", "Scaling", "CRUD", "Identity" ],
+            "color": "rgb(186, 211, 245)",
+            "createFlowForCategory": false,
```
Comment (nmallick1) [inline AngularApp/projects/app-service-diagnostics/src/armResourceConfig/microsoft.containerservice/managedclusters/config.json]: Please change this flag to true. Although it is set to true in override.json so there won't be any functionality issues, however the correct value for this flag is always true.

Comment (yunjchoi) [reply AngularApp/projects/app-service-diagnostics/src/armResourceConfig/microsoft.containerservice/managedclusters/config.json]: Thanks Nazeef, I changed the value to true.

**Pull request merged**